VagrantFile

```diff
@@ -0,0 +1,65 @@
+#to UNIX EOL
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+# All Vagrant configuration is done below. The "2" in Vagrant.configure
+# configures the configuration version (we support older styles for
+# backwards compatibility). Please don't change it unless you know what
+# you're doing.
+
+Vagrant.configure(2) do |config|
+  # The most common configuration options are documented and commented below.
+  # For a complete reference, please see the online documentation at
+  # https://docs.vagrantup.com.
+
+  # Every Vagrant development environment requires a box. You can search for
+  # boxes at https://atlas.hashicorp.com/search.
+  config.vm.box = "generic/debian10"
+  config.vm.box_version = "3.1.10"
+  config.vm.define "lever-for-change"
+
+  # Disable automatic box update checking. If you disable this, then
+  # boxes will only be checked for updates when the user runs
+  # `vagrant box outdated`. This is not recommended.
+  config.vm.box_check_update = true 
+
+  # Create a forwarded port mapping which allows access to a specific port
+  # within the machine from a port on the host machine. In the example below,
+  # accessing "localhost:8080" will access port 80 on the guest machine.
+
+  # Create a private network, which allows host-only access to the machine
+  # using a specific IP.
+  config.vm.network "private_network", ip: "192.168.33.11"
+  config.vm.network "forwarded_port", guest: 80, host: 80
+  config.vm.network "forwarded_port", guest: 5000, host: 5000
+
+  # Share an additional folder to the guest VM. The first argument is
+  # the path on the host to the actual folder. The second argument is
+  # the path on the guest to mount the folder. And the optional third
+  # argument is a set of non-required options.
+  config.vm.synced_folder "../torque-sites", "/data/www/torque-sites", owner: "vagrant", group: "www-data"
+
+  # Provider-specific configuration so you can fine-tune various
+  # backing providers for Vagrant. These expose provider-specific options.
+  # Example for VirtualBox:
+  #
+  config.vm.provider "virtualbox" do |vb|
+    vb.customize ["modifyvm", :id, "--ioapic", "on"]
```

Comment:
I don't know much about VirtualBox, and maybe if I did these config options would make more sense.  I leave it to your judgement whether additional commenting is needed here; I just wanted to point out that the meanings of a couple of the settings here were not obvious to me.

Reply:
Honestly these are coming directly from the settings that the Permanent team put forward in theirs.  This setting will "Enable I/O APIC for the motherboard of the VM."

I took this to be an "if it ain't broke don't fix it" type of setting.

Similarly the memory and cpus seemed fine and not needing modifiation until / unless it did.

(Since they seem fairly vanilla / not specifically here to solve a problem related to our use I don't think a comment is needed.)